test/minikube/minikube_domain_resync_test.go

```diff
@@ -295,6 +298,10 @@ func TestAdminScaleDown(t *testing.T) {
 }
 
 func TestDomainResync(t *testing.T) {
+	if os.Getenv("NUODB_LICENSE") != "ENTERPRISE" {
+		t.Skip("Cannot test resync without the Enterprise Edition")
```

Comment:
I think the actual reason we need EE is that this test starts multiple SMs. Is this now the only test that only runs internally?

Reply:
yeah. Its the only one left. If we rewrote it in a way where it does no need 2 SMs, we could make it run in travis.

Reply:
That is impossible because we are exercising SM scaledown and we cannot delete the only archive object in a database.

Reply:
that's what I thought. It will hence remain the only test that we only run in bamboo. I am sure that we will accumulate more over time. I will resolve the comment.